lib/active_merchant/billing/gateways/paybox_direct.rb

```diff
@@ -125,6 +133,11 @@ def add_reference(post, identification)
         post[:numtrans] = identification[10,10]
       end
 
+      def add_amount(post, money, options)
+        post[:montant] = ('0000000000' + (money ? amount(money) : ''))[-10..-1]
```

Comment:
What does this change have to do with the currency? And what are all these `0`s for?